### PR TITLE
Adding interactive consent to blocks.create

### PR DESCRIPTION
**Pull request by abchatra**
- Removing debouncing as appinsights already does debouncing.
- Adding interactive consent to activity.edit & blocks.create. (PLEASE Review this)

### CODE REVIEW

File: webapp/src/app.tsx [hunk 1]
```diff
@@ -822,7 +822,7 @@ export class ProjectView
         let changeHandler = () => {
             if (this.editorFile) {
                 if (this.editorFile.inSyncWithEditor)
-                    pxt.tickActivity("edit", "edit." + this.editor.getId().replace(/Editor$/, ''))
+                    pxt.tickEvent("activity.edit", { editor: this.editor.getId().replace(/Editor$/, '')}, { interactiveConsent: true } );
```
Comment (jwunderl) [inline webapp/src/app.tsx:825]: This one gets hit in the arcade share page after pressing the `show code` (doesn't get hit in micro:bit immediately); that seems valid as it comes after the user has clicked the button to interact, but just want to double check

**Review by jwunderl — APPROVED** [review]
LGTM, one question on a tick firing in the share page but that's just double checking

File: webapp/src/app.tsx (hunk 1 above)
Comment (abchatra) [reply webapp/src/app.tsx:825]: Good point. Seems ok to me.

**Comment by abchatra** [comment]
@riknoll ?

**Comment by riknoll** [comment]
blocks.create is not necessarily triggered by the user's interaction, i think there might be instances where it is triggered by things like loading a tutorial (but i could be wrong)

**Comment by abchatra** [comment]
I have validated in arcade & minecraft in loading tutorials scenario (example: https://minecraft.makecode.com/#tutorial:/hour-of-code/glass) that blocks.create is only invoked when the block is dragged. Merging this.

**Pull request merged**